drivers/sensor/lps22hh/lps22hh_i2c.c

```diff
@@ -0,0 +1,52 @@
+/* ST Microelectronics LPS22HH pressure and temperature sensor
+ *
+ * Copyright (c) 2019 STMicroelectronics
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Datasheet:
+ * https://www.st.com/resource/en/datasheet/lps22hh.pdf
+ */
+
+#include <string.h>
+#include <i2c.h>
+#include <logging/log.h>
+
+#include "lps22hh.h"
+
+#ifdef DT_ST_LPS22HH_BUS_I2C
+
+static u16_t lps22hh_i2c_slave_addr = DT_INST_0_ST_LPS22HH_BASE_ADDRESS;
+
+#define LOG_LEVEL CONFIG_SENSOR_LOG_LEVEL
+LOG_MODULE_DECLARE(LPS22HH);
+
+static int lps22hh_i2c_read(struct lps22hh_data *data, u8_t reg_addr,
+				 u8_t *value, u16_t len)
+{
+	return i2c_burst_read(data->bus, lps22hh_i2c_slave_addr,
+			      reg_addr, value, len);
+}
+
+static int lps22hh_i2c_write(struct lps22hh_data *data, u8_t reg_addr,
+				  u8_t *value, u16_t len)
+{
+	return i2c_burst_write(data->bus, lps22hh_i2c_slave_addr,
+			       reg_addr, value, len);
+}
+
+lps22hh_ctx_t lps22hh_i2c_ctx = {
```

Comment:
Why isn't this part of `struct lps22hh_data`?

Reply:
moved in data